source/_integrations/zwave_js.markdown

```diff
@@ -966,3 +966,28 @@ In Home Assistant, it is currently not possible to set the Z-Wave controller int
 ### Including / excluding a controller in an existing network using [classic inclusion](#classic-inclusion-versus-smartstart)
 
 A Z-Wave controller that manages an empty network can also join a different network and act as a secondary controller there. However, with Home Assistant, this is not possible. Home Assistant does not allow the Z-Wave controller to join another network, because Home Assistant acts as the central hub.
+
+## Z-Wave Command Classes supported in Home Assistant
+
+The following table lists Command Classes that are supported in Home Assistant, together with their version and their required
+Security Class.
```

Comment:
This is tricky...

In Z-Wave terms, a **supported** command class (CC) means that another Z-Wave device can send HA controlling commands of the same CC, and will get a correct response. Those have to be documented.

However users likely understand something else under **supported**, namely which CCs they can interact with in their devices. That list is much, much longer, does not have to be documented (and IMO should not).

I think we need to distinguish between those meanings here. I'm not 100% sure how though. Maybe
 ```suggestion
The following table lists Command Classes that Home Assistant will respond to when queried by other devices, together with the implemented version and required security class:
```